packages/desktop-client/src/components/autocomplete/FilterList.tsx

```diff
@@ -30,6 +25,7 @@ function FilterList<T extends { id: string; name: string }>({
           ...(!embedded && { maxHeight: 175 }),
         }}
       >
+        <Fragment>{ItemHeader({ title: 'Saved Filters' })}</Fragment>
```

Comment:
We should pass a type here so we aren't using `undefined`

Also, if we aren't using a key, should we just use <> for a fragment?

And is a fragment required here for ItemHeader?

Reply:
Fragment isn't needed. Fixed it.